data_structures/priority_queue.cpp

```diff
@@ -0,0 +1,170 @@
+/**
+ * @file
+ * @brief Implemetation of Max Priority Queue uning max-heap
+ * More about it: https://en.wikipedia.org/wiki/Priority_queue
```

Comment:
Markdown link formatting should be used here.
```suggestion
 * @brief Implementation of Max [Priority Queue](https://en.wikipedia.org/wiki/Priority_queue) using max-heap
```